app/assets/stylesheets/content/_tooltips.md

```diff
@@ -107,6 +107,20 @@ These can contain simple texts but are not suitable for HTML within the Tooltip.
 
 Note that the tabindex has to be set manually on the `<span>` and not the containing element. `tabindex="0"` makes the item tabbable at all.
 
+### Html tooltips
```

Comment:
it's `HTML`, not `Html`

(Sorry, that one is just me letting out my inner neat freak)